scripts/mono-repo-prep.js

```diff
@@ -102,13 +102,22 @@ const moveFiles$ = (package) => {
     const mvReadme$ = () => exec$(moveCmd('readme.md'))
     const mvTSConfig$ = () => exec$(moveCmd('tsconfig.json'))
     const mvStencilConfig$ = () => exec$(moveCmd('stencil.config.ts'))
+    const mvSrc$ = () => exec$(moveCmd('src'))
+    const mvUtils$ = () => exec$(moveCmd('utils'))
+    const mvStorybook$ = () => exec$(moveCmd('.storybook'))
```

Comment:
Do we need to move these ? I think these will live in the root dir of the mono-repo and then just referencing all the stories ion packages ect..